variables.tf

```diff
@@ -155,19 +214,32 @@ variable "cur_report_frequency" {
     condition     = can(regex("^(DAILY|HOURLY|MONTHLY|NONE)$", var.cur_report_frequency))
     error_message = "Valid options: DAILY, HOURLY, MONTHLY, NONE"
   }
-
 }
 
 #
 # SCPs & OUs
 variable "service_control_policies" {
   description = "Map of SCPs to deploy"
   default     = {}
+  type = map(
+    object({
+      policy_name        = string
+      policy_description = string
+      policy_json_file   = string
+      policy_targets     = optional(list(string))
+    })
```

Comment:
We're missing `policy_vars` which is an `optional(map(string))`

Example usage in the tfvars:
```hcl
    suspended_ou = {
      policy_name        = "SuspendedAccounts"
      policy_description = "Denies all activity in accounts in the SuspendedOU"
      policy_json_file   = "policies/SuspendedAccountsPolicy.json.tftpl"
      policy_targets     = ["Suspended"]
      policy_vars = {
        audit_role_name = "pht-audit"
      }
    }
```

Reply:
Added the `optional` `map`.